File: Day25/pyramid_pattern.c
#include<stdio.h>

//Function to print Pyramid pattern
void print_pyramid(int n){
    for(int i=0; i<n; i++){

        // Print space before asterisks
        for(int j = 0; j<n-i-1; j++){
            printf(" ");
        }
        // Print asterisks
        for(int k =0; k<2*i+1; k++){
            printf("*");
        }
        printf("\n");
    }
}
int main(){
    int n;
    printf("Enter the value of n: ");
    scanf("%d",&n);
    print_pyramid(n);
    return 0;
}

File: Day25/pyramid_pattern.py
"""Program for centered pyramid pattern"""
def print_pyramid(n):
    """ Function to print Pyramid pattern"""
    for i in range(n):

        # Print space before asterisks
        for _ in range(n-1-i):
            print(" ",end='')

        # Print asterisks
        for _ in range(2*i+1):
            print("*",end='')

        print()

row = int(input("Enter the number of rows : "))

print_pyramid(row)
